Marquee/ViewControllers/SearchViewController.swift

```diff
@@ -75,8 +75,9 @@ extension SearchViewController: UICollectionViewDelegate, UICollectionViewDataSo
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = searchCollectionView.dequeueReusableCell(withReuseIdentifier: CollectionViewCell.identifier, for: indexPath) as! CollectionViewCell
         if let posterPath = searchData?[indexPath.row].poster_path {
-            let url = URL(string: "https://image.tmdb.org/t/p/w500/\(posterPath)") ?? URL(string: "https://image.tmdb.org/t/p/w500/jrgifaYeUtTnaH7NF5Drkgjg2MB.jpg")
-            cell.poster.kf.setImage(with: url)}
+            let url = URL(string: "https://image.tmdb.org/t/p/w500/\(posterPath)")
```

Comment:
It can be useful to keep your raw strings in a constant struct. Example:
```
struct Constant {
        struct URLs {
               static let imageUrl = "https://image.tmdb.org/t/p/w500/\"
        }
}

let url = URL(string: Constant.URLs.imageUrl + posterPath)
```